### PR TITLE
add tag "Lists"

### PR DESCRIPTION
**Pull request by alexhough**
also remove Lists from caption

### CODE REVIEW

File: editions/tw5.com/tiddlers/wikitext/Lists in WikiText.tid [hunk 1]
```diff
@@ -1,7 +1,7 @@
-caption: Lists
```
Comment (pmario) [inline editions/tw5.com/tiddlers/wikitext/Lists in WikiText.tid]: Should not be removed. Is needed!

**Comment by sukima** [comment]
### The what

add tag "Lists" also remove Lists from caption

### How

https://github.com/Jermolene/TiddlyWiki5/pull/3146/files

### Why

But where's the **why**? Why is `Lists` removed from Caption? We want "Lists in WikiText" in the TOC? What other tiddlers are tagged as a `Lists`?

Don't get me wrong. I'm not against the change I was only illustrating that a key component (which is the reasoning behind the change) is missing.

**Comment by pmario** [comment]
OK

**Comment by alexhough** [comment]
**The Why?**

The concept of listing is such an important part of organising. 

I think we should have a new tiddler with the title "Lists"

Removing the caption "Lists" from Lists in Wikitext paves the way for this

**Comment by alexhough** [comment]
The convention where the caption field is used to simplify the title of the widgets is causing inconsistencies, see below



screen shot 2018-03-04 at 07 08 00


Another way of listing the core macros without the suffix would be to use a filter.

There could be a variant of the tree Macro which could take a suffix:

`<<tree suffix:"Macro">>`

**Pull request closed without merging**